eng/Sign.proj

```diff
@@ -36,7 +36,7 @@
     </ItemGroup>
 
     <ItemGroup Condition="'$(SignNugetPackages)' == 'true'">
-      <FilesToSign Include="$(OutDir)*.nupkg" Exclude="$(OutDir)*.symbols.nupkg">
+      <FilesToSign Include="$(OutDir)*.nupkg">
```

Comment:
I think we should also sign snupkg, right? Since the plan is for it to go into nuget? @eerhardt what do you think?

Reply:
Opps we merged right at the same moment I commented on this, but @imback82 I think we want to solve this.

Reply:
I guess it depends on what we plan on doing with them... if we just use it to publish the symbols to the Microsoft Symbol Server during the build, they probably don't need to be signed.
But if we do plan on uploading them to nuget.org, then maybe they need to be signed?

@dagood @mikem8361 - for thoughts/recommendations on what should be done with `.snupkg` files.

Reply:
@tmat knows the most about `snupkg`s, I think.

Reply:
snupkgs should be published to NuGet.org **and** the PDBs to Microsoft Symbol servers.